src/test/java/com/znsio/rpap/BaseTest.java

```diff
@@ -0,0 +1,51 @@
+package com.znsio.rpap;
+
+import com.znsio.api.ApplitoolsInitializer;
+import com.znsio.rpi.properties.Config;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+import com.znsio.rpap.pages.Page;
+import com.znsio.rpap.utils.DriverFactory;
+
+import java.time.Duration;
+import java.util.Properties;
+
+import static com.znsio.rpi.utils.ReportPortalLogger.logInfoMessage;
+
+public class BaseTest extends ApplitoolsInitializer {
+    protected static WebDriver webDriver;
+    private static WebDriverWait wait;
+    private static final Logger LOGGER = Logger.getLogger(BaseTest.class.getName());
+    private static final Properties config = Config.loadProperties(System.getProperty("CONFIG"));
+    protected Page page;
+
+    @BeforeSuite
+    public void SuiteSetup() {
```

Comment:
This may be a good place to provide custom UFG configuration for web tests

Reply:
Let me think. Can I also add this work as improvement for Raghav/Megha to implement?